tests/system/action/user/test_forget_password.py

```diff
@@ -0,0 +1,73 @@
+from time import time
+
+from openslides_backend.action.mixins.send_email_mixin import (
+    ConnectionSecurity,
+    EmailSettings,
+)
+from tests.system.action.base import BaseActionTestCase
+from tests.system.action.mail_base import AIOHandler, AiosmtpdServerManager
+
+
+class UserForgetPassword(BaseActionTestCase):
```

Comment:
Please add a test with a non-existent email address. The request should succeed just like a test with an existing email, but no email should actually be sent (so an attacker cannot filter which email addresses exist and which do not).

Reply:
Add a test.